java/core/src/main/scala/com/azavea/pdal/Main.scala

```diff
@@ -0,0 +1,54 @@
+package com.azavea.pdal
+
+import collection.JavaConverters._
+
+/** Test object should be removed from this PR and replaced with tests */
+
+object Main {
+  val iter: Iterator[Int] = Set(2).toIterator
```

Comment:
Glad to see these manual tests run, now's the time to start pulling them out into unit tests IMO